dipy/reconst/ivim.py

```diff
@@ -590,7 +600,7 @@ def fit(self, data, bounds_de=None):
         x_f = self.x_and_f_to_x_f(x, f)
 
         # Setting up the bounds for least_squares
-        bounds = ([0.01, 0.005, 10**-4], [0.3, 0.02,  0.003])
+        bounds = BOUNDS_VP
```

Comment:
It looks like it's overwriting the input (from the keyword argument). Shouldn't this only be set from the global variable if `bounds is None` ?